cli/src/semgrep/test.py

```diff
@@ -476,10 +476,14 @@ def generate_test_results(
 
     config_missing_tests_output = [str(c[0]) for c in config_without_tests]
 
+    # in a test context, we don't want to honor the paths: (include/exclude)
```

Comment:
Maybe this comment should be right on top of `respect_rule_paths=False,`?

Reply:
yes ... I'm not in office but I've merged without the comment fix because other PR will fail because semgrep-rules changed.